examples/erc1155/lib.rs

```diff
@@ -264,7 +266,9 @@ mod erc1155 {
         /// Initialize a default instance of this ERC-1155 implementation.
         #[ink(constructor)]
         pub fn new() -> Self {
-            Default::default()
+            ink_lang::codegen::initialize_contract(|_| {
+                Contract::default();
+            })
```

Comment:
@Robbepop I found that this wasn't necessary. I could replace

```suggestion
            Default::default()
```

And I could deploy and use the contract fine. It seems like a coincidence since the default `ROOT_KEY` is `0x00` and that's how a default `Mapping` would end up initialized. Any thoughts on this?